plugins/CoreAdminHome/lang/en.json

```diff
@@ -98,8 +98,8 @@
         "UseCustomLogo": "Use a custom logo",
         "ValidPiwikHostname": "Valid Matomo Hostname",
         "WithOptionalRevenue": "with optional revenue",
-        "YouAreOptedIn": "You are currently opted in.",
         "YouAreOptedOut": "You are currently opted out.",
+        "YouAreNotOptedOut": "You are currently not opted out.",
```

Comment:
I think we can just say "You are not opted out."?

Reply:
good point, will update